addons/main/config.cpp

```diff
@@ -586,18 +586,6 @@ class CfgMods {
     };
 };
 
+#include "CfgDependencies.hpp"

```

Comment:
Should be `CfgSettings` It's easier to find if we name things how they are actually named. and could prevent headaches when we want to add more stuff to CfgSettings later, for whatever reason.